helix-term/src/main.rs

```diff
@@ -66,6 +66,7 @@ FLAGS:
     -V, --version                  Prints version information
     --vsplit                       Splits all given files vertically into different windows
     --hsplit                       Splits all given files horizontally into different windows
+    -w, --working-path <path>      Specify an initial working directory
```

Comment:
```suggestion
    -w, --working-dir <path>      Specify an initial working directory
```

I think calling working-diretocry is the more common naming convention (your documentation even calls it that) and more concise since that path can not be a a directory)